From: win7-support-bot <win7@cromite.org>
Date: Sun, 26 Jul 2026 00:00:00 +0000
Subject: WIN7: tp-ink / abseil

Files:
 third_party/ink/src/ink/strokes/stroke.cc

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
diff --git a/third_party/ink/src/ink/strokes/stroke.cc b/third_party/ink/src/ink/strokes/stroke.cc
index 6b1e666..8730ebb 100644
--- a/third_party/ink/src/ink/strokes/stroke.cc
+++ b/third_party/ink/src/ink/strokes/stroke.cc
@@ -150,7 +150,7 @@ void Stroke::RegenerateShape() {
   // Create thread local stroke shape resources to save allocations if
   // `thread_local` is supported, which is almost always. If not, fall back to a
   // regular local variable.
-#ifdef ABSL_HAVE_THREAD_LOCAL
+#if 0
   thread_local
 #endif
       ShapeGenerationResources shape_gen;
